crowbar_framework/app/models/batch/export.rb

```diff
@@ -17,6 +17,8 @@
 
 module Batch
   class Export < Base
+    attr_accessor :path
```

Comment:
but now that you mention it, shall I also make filename accessible? it is not used yet though...